ansible_runner/runner.py

```diff
@@ -532,8 +535,8 @@ def kill_container(self):
         container_name = self.config.container_name
         if container_name:
             container_cli = self.config.process_isolation_executable
-            cmd = '{} kill {}'.format(container_cli, container_name)
-            proc = Popen(cmd, stdout=PIPE, stderr=PIPE, shell=True)
+            cmd = ['{}'.format(container_cli), 'kill', '{}'.format(container_name)]
```

Comment:
I don't think formatting is necessary here at all.
```suggestion
            cmd = [container_cli, 'kill', container_name]
```